.github/workflows/build.yml

```diff
@@ -2,15 +2,35 @@ name: Build Code Samples
 
 on:
   push:
-    branches: [ release/3.* ]
+    branches: [ "release/3.[1-9]" ]
   pull_request:
-    branches: [ release/3.* ]
+    branches: [ "release/3.[1-9]" ]
```

Comment:
Does 3.0 cause us actual problems?
Just in case we get up to 3.10 it's probably best to add * at the end.

Reply:
I think not actually, because as long as this is not actually in the 3.0 code branch then it shouldn't matter, let me change that.

Reply:
Good call, if it isn't needed, then pattern-matching on numbers can be a bit icky 🤢